lib/index.js

```diff
@@ -16,6 +16,6 @@ module.exports = {
   },
   processors: {
     // https://eslint.org/docs/developer-guide/working-with-plugins#file-extension-named-processor
-    '<noop>': noop,
+    'noop': noop,
```

Comment:
@NullVoxPopuli @patricklx do we need to export this processor as part of our public API? Is it useful for anyone to import individually?

Reply:
i thought it's necessary for eslint to find it?

Reply:
Not sure. I'll leave it for now with the new name. Hopefully this new name makes sense.